libraries/AP_AccelCal/AccelCalibrator.cpp

```diff
@@ -379,9 +379,6 @@ void AccelCalibrator::run_fit(uint8_t max_iterations, float& fitness)
         }
 
         num_iterations++;
-        if (fitness - last_fitness < 1.0e-9f) {
-            break;
-        }
```

Comment:
the Iterations are not that many that user will notice a significant delay between full run and limited runs, but if you feel like including that I don't have any issue with that.